broker/applications/operators/test/operators.BackupService.spec.js

```diff
@@ -199,6 +199,7 @@ describe('operators', function () {
         sandbox.restore();
         mocks.agent.getBackupLogs(fakeLogs);
         mocks.agent.lastBackupOperation(backup_state);
+        mocks.cloudProvider.auth(1);
```

Comment:
Due to change in the order of execution of UTs, the token was expired and so I have to add mock for authentication.